software/drivers/state_lookup.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+def get_states_from_json():
+    with open("states.json", "r") as f:

```

Comment:
should we use a config file or something similar to specify file paths?

Reply:
Yeah that would be useful.  You could also pass it as a command line arg, but having a config file makes more sense for the GUI.